ZelBack/src/services/fluxshareService.js

```diff
@@ -546,8 +547,8 @@ async function fluxShareRemoveFolder(req, res) {
 
       const dirpath = path.join(__dirname, '../../../');
       const filepath = `${dirpath}ZelApps/ZelShare/${folder}`;
-      await fs.promises.rmdir(filepath);
-
+      // await fs.promises.rmdir(filepath);
```

Comment:
why are we not using fs for it? instead of it it executes sudo rm -rf.

Reply:
fs.promises.rmdir will not remove not empty dir i want give ability to remove directory with files

Reply:
You can actually achieve that using FS module as its described in the docs
https://nodejs.org/api/fs.html#fspromisesrmdirpath-options

Reply:
but it will not solve situation when i want remove all file and folders but i want keep directory ( or i miss something)